### PR TITLE
changes to run oca-copy-maintainers on OCA v10

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check Travis

**Comment by gurneyalex** [comment]
This code has been running for some time on the oca server, but I had forgotten to push the changes back to the repository...

**Comment by pedrobaeza** [comment]
Can I squash commits?

File: tools/copy_maintainers.py [hunk 1]
```diff
@@ -88,46 +88,56 @@ def copy_users(odoo, team=None, dry_run=False):
...
+        print(u'Following users miss GitHub login:')
+        print(colors.FAIL +
+              '\n'.join(user.encode('utf-8')
+                        for user in no_github_login) +
               colors.ENDC)
 
     if not_found:
         print()
-        print('The following odoo projects have no team in GitHub:')
+        print(u'The following odoo projects have no team in GitHub:')
```
Comment (elicoidal) [inline tools/copy_maintainers.py]: Odoo

**Pull request merged**